boards/arm/nucleo_l476rg/Kconfig.defconfig

```diff
@@ -35,4 +35,7 @@ config PWM_STM32_2
 
 endif # PWM
 
+config I2C_1
```

Comment:
This line should be under `if SPI` statement.

Reply:
Added under "if I2C" instead.